sky/clouds/kubernetes.py

```diff
@@ -167,8 +188,10 @@ class Kubernetes(clouds.Cloud):
             ('Docker image is not supported in Kubernetes. ')
     }
 
-    IMAGE = 'us-central1-docker.pkg.dev/' \
-            'skypilot-375900/skypilotk8s/skypilot:latest'
+    IMAGE_CPU = ('us-central1-docker.pkg.dev/'
+                 'skypilot-375900/skypilotk8s/skypilot:latest')
+    IMAGE_GPU = ('us-central1-docker.pkg.dev/skypilot-375900/'
+                 'skypilotk8s/skypilot-gpu:latest')
```

Comment:
Ah that's a good idea... since this would require adding a service catalog skeleton for Kubernetes, I'm thinking of doing it in a separate PR. I have filed #2430 to track this